src/ci/docker/x86_64-gnu/Dockerfile

```diff
@@ -16,9 +16,13 @@ RUN apt-get update && apt-get install -y --no-install-recommends \
 COPY scripts/sccache.sh /scripts/
 RUN sh /scripts/sccache.sh
 
+ENV RUSTBUILD_FORCE_CLANG_BASED_TESTS yes
```

Comment:
Checking a [recent test run](https://travis-ci.com/rust-lang/rust/builds/97550307) this builder is already running too long at 2h11m, perhaps we can pick a faster builder like x86_64-gnu-debug to run these tests in?

Reply:
But that doesn't seem to run tests?